apps/website/src/app/docs/report/page.tsx

```diff
@@ -109,75 +256,99 @@ export default function RatingShieldsPage() {
+              ensName={getExampleReportName(undefined, true)}
+              hadLoadingError={true}
+            />
           </div>
           <div className="flex items-center justify-center">
-            <ReportBadge name="loading.eth" />
+            <ReportBadge ensName={getExampleReportName()} />
           </div>
         </div>
 
         <div className="grid grid-cols-6 gap-x-6 py-5">
           <div className="flex items-center">
             <div className="flex items-center font-mono">
-              <pre>{"<ReportBadge onClick />"}</pre>
+              <pre>{"<ReportBadge \n   onClickOverride />"}</pre>
             </div>
           </div>
 
           <div className="flex items-center justify-center">
             <ReportBadge
-              name="lightwalker.eth"
-              data={getReportExampleData(Rating.pass)}
-              onClick={() => alert("Clicked")}
+              ensName={getExampleReportName(Rating.pass)}
+              data={getExampleReportData(Rating.pass)}
+              onClickOverride={() => alert("Clicked")}
             />
           </div>
           <div className="flex items-center justify-center">
             <ReportBadge
-              name="culturecafé.eth"
-              data={getReportExampleData(Rating.warn)}
-              onClick={() => alert("Clicked")}
+              ensName={getExampleReportName(Rating.warn)}
+              data={getExampleReportData(Rating.warn)}
+              onClickOverride={() => alert("Clicked")}
             />
           </div>
           <div className="flex items-center justify-center">
             <ReportBadge
-              name="‍420.eth"
-              data={getReportExampleData(Rating.alert)}
-              onClick={() => alert("Clicked")}
+              ensName={getExampleReportName(Rating.alert)}
+              data={getExampleReportData(Rating.alert)}
+              onClickOverride={() => alert("Clicked")}
             />
           </div>
           <div className="flex items-center justify-center">
             <ReportBadge
               hadLoadingError={true}
-              name="notrab.eth"
-              onClick={() => alert("Clicked")}
+              ensName={getExampleReportName(Rating.pass, true)}
+              onClickOverride={() => alert("Clicked")}
             />
           </div>
           <div className="flex items-center justify-center">
-            <ReportBadge name="loading.eth" onClick={() => alert("Clicked")} />
+            <ReportBadge
+              ensName={getExampleReportName()}
+              onClickOverride={() => alert("Clicked")}
+            />
           </div>
         </div>
       </div>
     </div>
   );
 }
 
-const getReportExampleData = (rating: Rating): ConsolidatedNameGuardReport => {
+const getExampleReportName = (
+  rating?: Rating,
+  compWithError = false,
+): ENSName => {
+  if (compWithError) return buildENSName("notrab.eth");
+
+  if (!rating) return buildENSName("loading.eth");
+
+  switch (rating) {
```

Comment:
Better to call `buildENSName(getExampleReportData(rating).name)` or something like that.